active_learning_multiannotator/active_learning.ipynb

```diff
@@ -0,0 +1,419 @@
+    "\n",
+    "We also measure the model performance on a test set each round to demonstrate the improvement of the model."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 9,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "num_rounds = 15\n",
+    "num_examples_to_relabel = 100"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 10,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "model_accuacy_arr = np.full(num_rounds, np.nan)\n",
+    "\n",
+    "for i in range(num_rounds):\n",
+    "    # get consensus labels\n",
+    "    if i == 0:\n",
+    "        consensus_labels = get_majority_vote_label(multiannotator_labels)\n",
+    "    else:\n",
+    "        # we can use the pred_probs from last round as the best model estimate\n",
+    "        results = get_label_quality_multiannotator(multiannotator_labels, pred_probs_labeled, temp_scale=True)\n",
+    "        consensus_labels = results[\"label_quality\"][\"consensus_label\"]\n",
+    "\n",
+    "    # train model to get out-of-sample predicted probabilites and model accuracy for the current round\n",
+    "    pred_probs, pred_probs_unlabeled, model_accuacy = train_model(\n",
+    "        ExtraTreesClassifier(),\n",
+    "        X_labeled,\n",
+    "        consensus_labels,\n",
+    "        cv_n_folds=5,\n",
+    "        X_unlabeled=X_unlabeled,\n",
+    "        X_test=X_test,\n",
+    "        labels_test=true_labels_test,\n",
+    "    )\n",
+    "\n",
+    "    # compute active learning scores\n",
+    "    active_learning_score, active_learning_score_unlabeled = get_active_learning_scores(\n",
+    "        multiannotator_labels, pred_probs, pred_probs_unlabeled\n",
+    "    )\n",
+    "\n",
+    "    # get the indices of examples to collect more labels for\n",
+    "    relabel_idx, relabel_idx_unlabeled = get_idx_to_relabel(\n",
+    "        active_learning_score=active_learning_score,\n",
+    "        active_learning_score_unlabeled=active_learning_score_unlabeled,\n",
+    "        num_examples_to_relabel=num_examples_to_relabel,\n",
+    "    )\n",
+    "\n",
+    "    # format the data for the next round of active learning \n",
+    "    # (moving some unlabeled examples to the labeled pool because we are collecting labels for them)\n",
+    "    (\n",
+    "        multiannotator_labels, relabel_idx_combined, X_labeled, X_unlabeled, pred_probs_labeled, \n",
+    "        pred_probs_unlabeled, extra_labels_labeled, extra_labels_unlabeled,\n",
+    "    ) = setup_next_iter_data(\n",
+    "        multiannotator_labels, relabel_idx, relabel_idx_unlabeled, X_labeled, X_unlabeled, pred_probs, \n",
+    "        pred_probs_unlabeled, extra_labels_labeled, extra_labels_unlabeled,\n",
+    "    )\n",
+    "\n",
+    "    # add a new annotator that provides new labels for the examples with lowest active learning scores (indices obtained above)\n",
+    "    multiannotator_labels = add_new_annotator(\n",
```

Comment:
this makes it sound like the new batch of data must be annotated by a single new annotator, so try to think of how to make it clear that new batch of data can be labeled by multiple annotators (existing annotators could handle some of these examples as well)